activerecord/lib/active_record/scoping/default.rb

```diff
@@ -57,7 +57,7 @@ def before_remove_const #:nodoc:
         # default_scopes for the model  where +all_queries+ is true.
         def default_scopes?(all_queries: false)
           if all_queries
-            self.default_scopes.map(&:all_queries).include?(true)
+            self.default_scopes.any?(&:all_queries)
```

Comment:
I just realized that technically it will work a little bit differently because `.include?(true)` explicitly checks for a `true` object, when `.any?` will look for any "truthy" value, i.e.
```ruby
Obj = Struct.new(:all_queries)
data = [Obj.new("I'm just a truthy value"), Obj.new(false)]
data.any?(&:all_queries) #  => true
data.map(&:all_queries).include?(true) # => false
```

But I believe we are fine because it seems like by design the `all_queries` attribute is supposed to be either boolean or `nil`
We can do `self.default_scopes.any? { |s| s.all_queries == true }` to keep the exact behaviour but I feel like it would be an overkill